dask_match/tests/test_core.py

```diff
@@ -144,14 +149,26 @@ def test_conditionals(func):
     df["y"] = df.y * 2.0
     ddf = from_pandas(df, npartitions=10)
 
-    assert_eq(func(df), func(ddf))
+    assert_eq(func(df), func(ddf).compute())
+
+
+@pytest.mark.xfail(reason="TODO: Debug this")
+def test_predicate_pushdown(tmpdir):
+    from dask_match.io.parquet import ReadParquet as ReadPq
```

Comment:
Why this renaming?  It seems like it might make more sense to just use the `ReadParquet` class in `io.parquet`.  Having two class names like this seems like it complicates things to me.

Reply:
Oh, maybe this is coming from the difference of having a function and class with the same name.  I suspect that this will become simpler if we name the function `read_parquet` like in dask.dataframe.

Reply:
yes, exactly. This is another temporary thing - You were too quick :)